temporian/implementation/numpy/data/event_set.py

```diff
@@ -587,3 +585,76 @@ def creator(self) -> Optional[Operator]:
         created EventSets have a `None` creator.
         """
         return self.node()._creator
+
+    def _repr_html_(self) -> str:
```

Comment:
I see this function are growing significantly, especially when we add interactive elements. Could it be moved in a separate file ?